arduino/OpenMRNLite.h

```diff
@@ -362,6 +362,11 @@ class OpenMRN : private Executable
         haveExecutorThread_ = true;
         stack_->executor()->start_thread(
             "OpenMRN", OPENMRN_TASK_PRIORITY, OPENMRN_STACK_SIZE);
+#ifdef ESP32
+        // Remove IDLE0 task watchdog, because the openmrn task sometimes gets
+        // scheduled on CPU0 and can use 100% cpu.
+        esp_task_wdt_delete(xTaskGetIdleTaskHandleForCPU(0));        
```

Comment:
switch to ```disableCore0WDT();``` which is the API from arduino-esp32 that is recommended for this sort of usage. we can alternatively do this for CORE 1 if the executor ends up there instead.

Reply:
there is no WDT on core 1 by default in arduino (since the loop task is there the idle is never run by default)

Reply:
WDT can be enabled on core 1 by the user. You are correct though that it is off by default. The idle task will still run regardless though for general housekeeping tasks but it should be minimal impact.